modules/users/tests/server/user.server.routes.tests.js

```diff
@@ -742,7 +742,7 @@ describe('User CRUD tests', function () {
               }
 
               // Call the assertion callback
-              userInfoRes.body.message.should.equal('Username already exists');
+              userInfoRes.body.message.toLowerCase().should.containEql('username already exists');
```

Comment:
LGTM, but can you add a comment saying that this works for x versions of MongoDB.